.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dtolnay/rust-toolchain@stable
+      - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84
```

Comment:
nit: not sure how it is right now, but up until some time ago in order for dependabot to work you needed to have a tag information in the same lne after the comment. I guess that even if this works without it, it may be a good idea to add this information so people know straight away what version we're using ;) 

to be clear, I mean something like:

```
- uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84 # v1.2.3
```